pkg/plugins/plugins.go

```diff
@@ -63,6 +64,7 @@ type Plugin struct {
 	SecretsManager secretsmanagerplugin.SecretsManagerPlugin
 	client         backendplugin.Plugin
 	log            log.Logger
+	Trace          tracing.Tracer
```

Comment:
If I don't make it exported, `cmp.Equal` throws an error() in `loader_test.go` -> `Load a Core plugin`: